packages/csv-parser-discount-code/test/utils.spec.js

```diff
@@ -0,0 +1,58 @@
+import castTypes from '../src/utils'
+
+describe('castTypes', () => {
+  test('should properly cast object types', () => {
```

Comment:
Because they mean (and do) the same thing... And you could have as well asked me the reverse question if I used `it`... (Besides, that's more like Integration test 😜)